### ISSUE TITLE
Estraneous characters in Miranda NG when using IRC

### ISSUE THREAD

**Issue by sukiletxe** [nvaccess/nvda#6200]
Steps to reproduce:

* 1. Add a IRC account in Miranda NG.
* 2. Open the server window or chat in a channel.
    * 2.1. Read line by line with arrows. (Notice the straneous letters before the message).
    * 2.2. Now, read character by character using arrows. (See, they're gone. A single one shows up, probably a ~ or a y.)
    * 2.3. Read using review mode, both character by character, and line by line. (Same azs 2.1)

**Comment by Piciok** [comment]
Hello Sukil,

This might not be a NVDA issue, rather a setting in your Miranda chat window options.

What plugin are you using for your chats? I have observed such behaviour with Scriver, may be happening with others too, when the option "Show message icons" is selected in Options -- Message Session -- Message log.

Make sure this is disabled and see if the problem still persists.

Hope it helps.

kind regards

Paweł
  ----- Original Message ----- 
  From: Sukil Etxenike 
  To: nvaccess/nvda 
  Sent: Thursday, July 21, 2016 11:46 PM
  Subject: [nvaccess/nvda] Estraneous characters in Miranda NG when using IRC (#6200)


  Steps to reproduce:

    a.. 1. Add a IRC account in Miranda NG. 
    b.. 2. Open the server window or chat in a channel. 
      a.. 2.1. Read line by line with arrows. (Notice the straneous letters before the message). 
      b.. 2.2. Now, read character by character using arrows. (See, they're gone. A single one shows up, probably a ~ or a y.) 
      c.. 2.3. Read using review mode, both character by character, and line by line. (Same azs 2.1)

**Comment by sukiletxe** [comment]
Oh, I think I'm not able to disable it. I don't know why. Perhaps 
because the checkbox is not exposed properly (I have tried with the 
mouse too, but no rsults).

**Comment by Piciok** [comment]
Try to do it blindly and see if the result is achieved. These checkboxes do not expose their state properly but I think pressing the spacebar on them does actually change their state.
  ----- Original Message -----

**Comment by sukiletxe** [comment]
No, it doesn't. Or maybe that isn't actually the setting. I don't notice any difference.

**Comment by Adriani90** [comment]
@sukiletxe can you test if this is still an issue in last Miranda NG version? I think it is 0.95 or so.

**Comment by Adriani90** [comment]
This is still an issue, it seems NVDA reports some symbols code in front of every message when navigating with up and down arrow through the message history.
For example
+0000000003E7E0E8
and then the time stamp followed by the message.
This seems to occur only on IRC accounts, not on Facebook ro so. I think this might be specific to IRC formatting styles.
cc: @codeofdusk

**Comment by seanbudd** [comment]
Is this still an issue?
